Modules/Core/Common/test/itkSimpleFilterWatcherTest.cxx

```diff
@@ -31,20 +31,12 @@ class TanHelper
   TanHelper() = default;
   ~TanHelper() = default;
   bool
-  operator!=(const TanHelper & rhs) const
+  operator==(const TanHelper & rhs) const
   {
-    return this != &rhs;
+    return this == &rhs;
   }
 
-  bool
-  operator==(const TanHelper & other) const = delete;
```

Comment:
@hjmjohnson I see you added the `= delete` here almost three years ago, hope it's OK to you that this PR actually removes the `= delete`! I believe it's in general preferable for a class to define `operator==` (without `= delete`) whenever it defines `operator!=`. To simply make it an "equality comparable" type. (C++20 will automatically generate `operator!=` for a class that has `operator==`.)
https://github.com/InsightSoftwareConsortium/ITK/commit/dbcdf4bce2cf4e0e31526064f823a5ccd4454293#diff-8ff9e0510254bcf8e71ee15fa988a4eb4ea1f83b6a896bed25caa273108c8d6c

Reply:
@N-Dekker Your work here is great!  Thanks for updating and making the code consistent.  I don't know the complete history of why these were not implemented.  The "= delete" was a refactoring to explicitly remove use of functions that were previously not implemented.

Having proper implementations seems like a great idea.

Hans